ansible/roles/custom_mounts/tasks/install_packages.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Common Ceph setup block
+  vars:
+    has_ceph_mount: >-
+      {{ custom_mounts | dict2items | selectattr('value.type', 'in', ['cephfs', 'ceph_rbd']) | list | length > 0 }}
+  when: has_ceph_mount
+  become: true
+  block:
+    - name: Gather installed package facts
+      ansible.builtin.package_facts:
+        manager: auto
+
+    - name: Install epel-release and Ceph repo if ceph-common is not installed
+      ansible.builtin.dnf:
+        name:
+          - epel-release
+          - "centos-release-ceph-{{ ceph_repo_release }}"
```

Comment:
Need to check if this is compat with what we've got from stackhpc.os-manila-mount